pgctl/service.py

```diff
@@ -150,16 +165,36 @@ def ensure_exists(self):
         if not self.path.check(dir=True):
             raise NoSuchService("No such service: '%s'" % bestrelpath(str(self.path)))
 
-        # ensure symlink {service_dir}/supervise -> {scratch_dir}/supervise
-        # this will re-connect the service to its state descriptors if the symlinks have been deleted or moved
-        supervise_in_scratch = self.scratch_dir.join('supervise')
-        supervise_in_scratch.ensure_dir()
-        symlink_if_necessary(supervise_in_scratch, self.path.join('supervise'))
+        self._ensure_supervise_is_scratch('supervise')
         self.__exists = True
 
     def ensure_logs(self):
         self.ensure_exists()
-        self.path.ensure('log')
+        self.path.ensure_dir('logs')
+        self.path.join('logs').ensure('current')
+
+        self.path.ensure_dir('log')
```

Comment:
i sort of like the idea of hiding as many autogenerated files as possible away in scratch, so that it's more obvious to users that (as much as possible) everything in the playground directory is there for them to edit

i'm okay with shipping this as-is though